### DISCUSSION TITLE
Wanted to ask how to switch NoteTpye mid song? useing playstate.hx or modchart.lua

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2712] [Q&A]
i want to change noteType like in Discharge or like mods that do change NoteType mid song i wanted to know how please.

**Comment by libracrumbs** [comment]
IIRC you should just be able to download the sc, and take it

**Reply by glowsoony** [reply]
wdym?!! IIRC? whats that and wdym sc?

**Reply by backhop** [reply]
> wdym?!! IIRC? whats that and wdym sc?

"if i remember correctly" and "source code"

**Reply by glowsoony** [reply]
wdym? WHAT SOURCE CODE? I WORK FOR A MOD SO THAT's WHAT I AM ASKING BRUH I HAVE MY OWN SOURCE CODE.
